crates/bevy_asset/src/path.rs

```diff
@@ -130,7 +130,7 @@ impl AssetPathId {
 
 /// this hasher provides consistent results across runs
 pub(crate) fn get_hasher() -> AHasher {
-    AHasher::new_with_keys(42, 23)
+    RandomState::with_seeds(42, 23, 13, 8).build_hasher()
```

Comment:
It is possible those seeds are not very secure. But then they weren't before, so...